src/code/z_vimode.c

```diff
@@ -1,23 +1,378 @@
 #include "global.h"
+#include "code/z_vimode/z_vimode.c"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_vimode/func_80140260.s")
+extern Gfx D_0E0001C8[];
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_vimode/func_8014026C.s")
+typedef struct {
+    /* 0x00 */ u32 unk_00;
+    /* 0x04 */ u32 unk_04;
+    /* 0x08 */ u32 unk_08;
+    /* 0x0C */ u32 unk_0C;
+    /* 0x10 */ u32 unk_10;
+    /* 0x14 */ u32 unk_14;
+    /* 0x18 */ u32 unk_18;
+} ViModeStruct; // size = 0x1C
```

Comment:
What is this a struct of? What kind of data is this organizing? Do you have a sense what these unknowns are?